src/util/system.cpp

```diff
@@ -135,6 +135,13 @@ bool DirIsWritable(const fs::path& directory)
     return true;
 }
 
+bool CheckDiskSpace(const fs::path& dir, uint64_t nAdditionalBytes)
+{
+    constexpr uint64_t nMinDiskSpace = 52428800;
```

Comment:
It would be good to add an additional comment above this indicating its approximate size. Previously this comment was present:

```
// Check for nMinDiskSpace bytes (currently 50MB)
```